### ISSUE TITLE
Block title (as modal title) double escaped when editing

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#1640]
Steps to reproduce:
- add a custom block to a region
- set the title to `Let's get in touch`
- save the block.
- configure the same block
- check the title of the modal, it will say `Configure block "Let&#039;s get in touch"`

**Comment by jenlampton** [comment]
It looks like we expect titles in modals to be double escaped, because we have some specific code in there to undo the double escaping:
```php
  // jQuery UI does not allow markup in its titles, and does its own escaping.
  // The value passed in should already be escaped (as it would be from
  // backdrop_get_title()), so we decode it back to the original characters.
  $title = html_entity_decode(strip_tags($title));
```

Unfortunately, it's not working. I think instead it needs to be this...
```php
  $title = html_entity_decode(strip_tags($title), ENT_QUOTES);
```

**Comment by quicksketch** [comment]
Looks great, thanks! Merged for 1.3.3.
